torch/jit/__init__.py

```diff
@@ -1731,6 +1731,31 @@ def __getstate__(self):
                 "Mixed serialization of script and non-script modules is not supported. " +
                 "For purely script modules use my_script_module.save(<filename>) instead.")
 
+        # Python magic methods do lookups on an object's class type, instead of looking up
+        # the desugared attribute. In order to continue to expose the magic methods
```

Comment:
nit: I was confused by the word "desguared" here. I think it would be clearer to talk about methods defined on classes vs. methods defined on instances.